### PR TITLE
Python 3.6 invalid escape sequence deprecation fixes

### PR DESCRIPTION
**Pull request by scop**
https://docs.python.org/3/whatsnew/3.6.html#deprecated-python-behavior

### CODE REVIEW

**Comment by scop** [comment]
Done.

File: git/remote.py [hunk 1]
```diff
@@ -391,7 +391,7 @@ def __init__(self, repo, name):
 
     def __getattr__(self, attr):
         """Allows to call this instance like
-        remote.special( \*args, \*\*kwargs) to call git-remote special self.name"""
+        remote.special( \\*args, \\*\\*kwargs) to call git-remote special self.name"""
```
Comment (isaacsanders) [inline git/remote.py]: Why not use a raw string here? (I am fairly new to Python, so this is a question seeking a real answer if there is an answer, not a passive aggressive question)

Comment (SethMichaelLarson) [inline git/remote.py]: Typically you don't see docstrings being raw strings.

Comment (isaacsanders) [inline git/remote.py]: Does it negatively effect their use as docstrings?

Comment (isaacsanders) [inline git/remote.py]: It seems like you might want to have the docstring be WYSIWYG, and the raw string appears to be meant for that, to me...

Comment (SethMichaelLarson) [inline git/remote.py]: I don't have any say in this repo so I don't mind either way. Just a preference and for homogeneity.

Comment (scop) [inline git/remote.py]: BTW, FWIW and all that, I don't know why the asterisks in this particular docstring are backslash escaped in the first place.

**Comment by Byron** [comment]
@scop I am very sorry for the delay, and thanks a lot for making this PR! I had a look at the link you provided, but was unable to understand the relation to this PR. Could you briefly explain the problem it is solving? My intuition is that the docstrings look incorrect when rendered.

**Comment by Byron** [comment]
Great, thanks a lot!

**Pull request merged**